### ISSUE TITLE
function does not appear in outline list if formal parameter list contains line break

### ISSUE THREAD

**Issue by gittar** [Hirse/brackets-outline-list#100]
**Details about your environment**

* **Brackets Version:** 
1.10 experimental build
* **Brackets Outline List Version:** 
1.4
**What did you do? Please include the actual source code causing the issue.**

```
def plot1(ax, X=None, C=None):
    pass

def plot2(ax, X=None, 
          C=None):
    pass
```

**What did you expect to happen? How did you expect the Outline to look?**
plot1 and plot2 appear in outline list.

**What actually happened? How did the Outline actually look?**
Only plot1 appeared in outline list.
Likely reason: The line break after "X=None,"

(the real example contains so many formal params that they do not fit in one line)

**Comment by Hirse** [comment]
Yes, that is the reason. Unfortunately based on the way functions are detected, this problem is not that easy to solve.
